### PR TITLE
CLI: uploadVideo - use relative paths when resolving endpoints

### PR DESCRIPTION
**Pull request by Lezek123**
Solves issues pointed out by @bwhm which arised when storage node base api url included some additional path, ie.:
```
http://localhost/storage/
```
In which case endpoints need to be resolved by using relative path, like:
```
http://localhost/storage/discover/v0/${storageProviderId}
```
Instead of absolute path:
```
http://localhost/discover/v0/${storageProviderId}
```

### CODE REVIEW

**Review by mnaamani — APPROVED** [review]
Will just wait for CI check to complete before merging.

**Review by bwhm — APPROVED** [review]

**Pull request merged**
